sherpa-onnx/csrc/online-recognizer-transducer-impl.h

```diff
@@ -183,6 +184,43 @@ class OnlineRecognizerTransducerImpl : public OnlineRecognizerImpl {
            s->NumFramesReady();
   }
 
+  // Warmping up engine with wp: warm_up count and max-batch-size
+  void WarmpUpRecognizer(int32_t warmup, int32_t mbs) const {
+    auto max_batch_size = mbs;
+    if (warmup <= 0 || warmup > 100)
+      return;
+    int32_t chunk_size = model_->ChunkSize();
+    int32_t chunk_shift = model_->ChunkShift();
+    int32_t feature_dim = 80;
+    std::vector<OnlineTransducerDecoderResult> results(max_batch_size);
+    std::vector<float> features_vec(max_batch_size * chunk_size * feature_dim);
+    std::vector<std::vector<Ort::Value>> states_vec(max_batch_size);
+
+    auto memory_info =
+        Ort::MemoryInfo::CreateCpu(OrtDeviceAllocator, OrtMemTypeDefault);
+
+    std::array<int64_t, 3> x_shape{max_batch_size, chunk_size, feature_dim};
+
+    // filling with 0.0, so that it doesn't affact the states,
+    // and left_context which is maintained by the encoders
+    std::fill(features_vec.begin(), features_vec.end(), 0.0f);
```

Comment:
```suggestion
```

`std::vector<float>` by default contains zeros on initialization, so there is no need to re-fill it with 0.

Reply:
ok got it